ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -36,18 +35,22 @@
 
   private static final Logger log = LoggerFactory.getLogger(CommandRunner.class);
 
+  private static final int STATEMENT_RETRY_MS = 1000;
```

Comment:
Rather than a fixed retry frequency, how about an exponential backoff, with some suitable upper limit?

Reply:
Yeah I think that's a good call.